kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -118,6 +130,18 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
     }
   }
 
+  def logAuditEvent(sql: String, state: OperationState): Unit = {
+    if (sql != null) {
+      val user = session.user
+      val address = session.ipAddress match {
+        case s if s != null => s
+        case _ => "unknown-ip-addr"
+      }
+
+      getAuditLog.get.info(s"user=$user\tip=$address\tstate=${state.name}\tsql=$sql\t")
```

Comment:
shall we log start time and elapsed time ? and also log app id I think it can help debug.